ravenpy/utilities/geoserver.py

```diff
@@ -6,7 +6,7 @@
 * BBox coordinates are passed as (lon1, lat1, lon2, lat2).
 * Shapes (polygons) are passed as shapely.geometry.shape parsable objects.
 * All functions that require a CRS have a CRS argument with a default set to WGS84.
-* GEO_URL points to the GeoServer instance hosting all files.
+* GEOSERVER_URL points to the GeoServer instance hosting all files.
```

Comment:
Same comment as the other https://github.com/Ouranosinc/raven/pull/485 about keeping backward-compat with the old public `GEO_URL`.

Reply:
This looks like an identical file as the other https://github.com/Ouranosinc/raven/pull/485, is this normal the same file is duplicated?